File: levels/level6/task.cpp
// Aufgabe: Schreibe ein Programm, das die Zahlen von 1 bis 10 ausgibt und anzeigt, ob jede Zahl gerade oder ungerade ist.
// Dein Code beginnt hier.

#include <iostream>

int main() {
    for (int zahl = 1; zahl <= 10; ++zahl) {
        if (zahl % 2 == 0) {
            std::cout << zahl << " ist gerade." << std::endl;
        } else {
            std::cout << zahl << " ist ungerade." << std::endl;
        }
    }
    return 0;
}

File: levels/level10/task.py
# Aufgabe: Schreibe ein Programm, das die Zahlen von 1 bis 10 ausgibt und anzeigt, ob jede Zahl gerade oder ungerade ist.
# Dein Code beginnt hier.

# Schleife über die Zahlen von 1 bis 10
for zahl in range(1, 11):
    # Bedingte Anweisung zur Überprüfung, ob die Zahl gerade ist
    if zahl % 2 == 0:
        print(zahl, "ist gerade.")
    else:
        print(zahl, "ist ungerade.")
